### DISCUSSION TITLE
Why is Inklecate 1.0.0+ a requirement?

### DISCUSSION THREAD

**Discussion by walterpalladino** [ephread/inkgd#77] [Q&A]
Someone let me understand why a pure GDScript implementation requires Inklecate 1.0.0+.
Thank You you in advance.

**Comment by francoisdlt** [comment]
it is not a requirement. you can build the JSON outside and run it in Godot without inklecate.
However, there is a nice feature where you can setup inklecate in the plugin and re-compile the JSON on the fly whenever a change is detected in your ink files. Very neat.

**Reply by walterpalladino** [reply]
Thank You for answer me.
